ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ViewScanConfigDetail.tsx

```diff
@@ -158,81 +152,41 @@ function ViewScanConfigDetail({
-                                                : 'Scan is in progress'}
-                                        </DescriptionListDescription>
-                                    </DescriptionListGroup>
-                                    <DescriptionListGroup>
-                                        <DescriptionListTerm>Last updated</DescriptionListTerm>
-                                        <DescriptionListDescription>
-                                            {getTimeWithHourMinuteFromISO8601(
-                                                scanConfig.lastUpdatedTime
-                                            )}
-                                        </DescriptionListDescription>
-                                    </DescriptionListGroup>
-                                </ScanConfigParametersView>
-                                <ScanConfigClustersTable
-                                    headingLevel={headingLevel}
-                                    clusterScanStatuses={scanConfig.clusterStatus}
-                                />
-                                <ScanConfigProfilesView
-                                    headingLevel={headingLevel}
-                                    profiles={scanConfig.scanConfig.profiles}
-                                />
-                                {isComplianceReportingEnabled && (
-                                    <NotifierConfigurationView
-                                        customBodyDefault={getBodyDefault(
-                                            scanConfig.scanConfig.profiles
-                                        )}
-                                        customSubjectDefault={getSubjectDefault(
-                                            scanConfig.scanName,
-                                            scanConfig.scanConfig.profiles
-                                        )}
-                                        notifierConfigurations={scanConfig.scanConfig.notifiers}
-                                    />
-                                )}
-                            </Flex>
-                        </CardBody>
-                    </Card>
-                )}
-            </PageSection>
+            {isReportJobsEnabled && (
+                <PageSection variant="light" className="pf-v5-u-py-0">
+                    <Tabs
+                        activeKey={selectedTab}
+                        onSelect={(_e, tab) => {
+                            setSelectedTab(ensureJobContextTab(tab));
+                        }}
+                        aria-label="Scan schedule details tabs"
+                    >
+                        <Tab
+                            tabContentId={configDetailsTabId}
+                            eventKey="CONFIGURATION_DETAILS"
+                            title={<TabTitleText>Configuration details</TabTitleText>}
+                            aria-label="Configuration details tab"
```

Comment:
Because `Tab` element renders **Configuration details** the `aria-label` prop might be redundant.

Can you comment out the pair of props and see if axe DevTools reports any issues?

Reply:
It didn't seem like anything changed when I removed it so maybe it is redundant. I'll remove it as well.